tests/reftests/env.test

```diff
@@ -53,3 +53,23 @@ OPAM_SWITCH_PREFIX=${BASEDIR}/OPAM/conffile
 ### opam exec --no-switch -- env | grep "^NV_VARS|^OPAM_SWITCH_PREFIX|${OPAM}"
 NV_VARS=/another/path
 OPAM=${OPAM}
+### : root and switch with spaces :
+### RT="$BASEDIR/root 2"
+### SW="switch w spaces"
+### OPAMNOENVNOTICE=0
+### opam init -na --bare --bypass-check --disable-sandbox --root "$RT" defaut ./REPO
+No configuration file found, using built-in defaults.
+
+<><> Fetching repository information ><><><><><><><><><><><><><><><><><><><><><>
+[defaut] Initialised
+### opam switch create "./$SW" nv --root "$RT"
+
+<><> Installing new switch packages <><><><><><><><><><><><><><><><><><><><><><>
+Switch invariant: ["nv"]
+
+<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+-> installed nv.1
+Done.
+# Run eval $(opam env --root='${BASEDIR}/root 2' --switch='${BASEDIR}/switch w spaces') to update the current shell environment
+### opam env --sw "./$SW" | grep OPAM_SWITCH_PREFIX | ';' -> ':'
```

Comment:
```suggestion
### opam env --sw "./$SW" | grep OPAM_SWITCH_PREFIX | ';' -> ':' | '\\_opam\'' -> '/_opam\''
```

Reply:
This should be handled by test engine, see https://github.com/ocaml/opam/blob/35899ba5c8d3dfa856076383ad5ae4ab64c66f4e/tests/reftests/env.test#L15-L16

Reply:
Anyway, this last test is not required for this PR (no extra escaping done in it)

Reply:
ah, then I don’t know how to fix this properly